packages/@aws-cdk/aws-codepipeline-actions/test/integ.lambda-deployed-through-codepipeline.lit.ts

```diff
@@ -1,3 +1,4 @@
+/// !cdk-integ:stack PipelineStack
```

Comment:
What does this do?

Reply:
it tells cdk-integ which stack to deploy